config/registry_config.go

```diff
@@ -44,17 +45,20 @@ func (*RegistryConfig) Prefix() string {
 	return constant.RegistryConfigPrefix
 }
 
-func loadRegistries(targetRegistries []string, registries map[string]*RegistryConfig, roleType common.RoleType) []*common.URL {
+func loadRegistries(targetRegistries string, registries map[string]*RegistryConfig, roleType common.RoleType) []*common.URL {
 	var urls []*common.URL
+	trSlice := strings.Split(targetRegistries, ",")
+
 	for k, registryConf := range registries {
 		target := false
 
 		// if user not config targetRegistries,default load all
-		if len(targetRegistries) == 0 {
+		// Notice:If s does not contain sep and sep is not empty, SplitAfter returns a slice of length 1 whose only element is s.
```

Comment:
what the meaning of s?

Reply:
oh，I copy the comment from strings.Split func. I will try to write a description.

Reply:
done